requirements.txt

```diff
@@ -5,7 +5,7 @@ click >= 6.0
 ecdsa >= 0.13
 behave
 protobuf
-pc_ble_driver_py >= 0.8.1
+pc_ble_driver_py >= 0.11.0
```

Comment:
To get the latest fix on pc-ble-driver-py (https://github.com/NordicSemiconductor/pc-ble-driver-py/commit/48d0f20), use 0.11.1.